lib/Alien/Base.pm

```diff
@@ -114,6 +114,8 @@ sub import {
   my $class = shift;
 
   return if $class->install_type('system');
+  
+  $class->dist_dir;
```

Comment:
It would be good to add a comment on why this line is called here (e.g., "Sanity check in order to ensure that dist_dir can be found. This will throw an exception otherwise.")